src/tests/run.py

```diff
@@ -1408,6 +1409,9 @@ def create_repro(args, env, tests):
     # Now that the args.repro_location exists under <runtime>/artifacts
     # create wrappers which will simply run the test with the correct environment
     for test in failed_tests:
+        if test["is_merged"]:
+            print("Skipping repro for merged test: %s (%s)" % (test["name"], test["assembly_display_name"]))
+            continue
         if test["test_path"] is None:
```

Comment:
Nit:
What about `elif test["test_path"] is None:`
than
```
    continue
if test["test_path"] is None:
```

Reply:
Yes, you right. The execution flow is the same but the code is simpler. I'm going to change it. Thanks!

Reply:
Done. Thanks!